notebooks/ids_detection/IDS_using_LODA.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<ul><li>might be nice if you explain why you selected these two metrics, </li><li>since you explain what (AP) is, maybe you can also add area under the curve for AUC</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>